.travis.yml

```diff
@@ -27,13 +28,15 @@ before_script:
 
 script:
   - make test
-  - make build
+  - make artifacts
 
 deploy:
   provider: releases
   api_key:
     secure: H514x+zUvUtT2xeJmqIanrMXtsvGroA/Bk5eAwNkVbz+2bL9Aj9tUSSpTvMka34lo+IdBDK0lpITDNooEmdlyCPRtJx/wAavSjKTquq+K0isXXCv4m/w7L3Gxhd5Xwq1ZEg4gB3cyQOoRTSLeopWqtx1EJ0gnF7MZL4IQ93zJL44YYBDVCp0mPoBOKmk6k6+vWhvmkHyepIGNriXM4kwm69mByrRdUb+YTyYl0wBKV7jW6zuoiWX2srFkArP1tyB4kc0Gc8+SIzoB1DykFw99cmN2H3DS5acG6omindaWE1izFodsv/Ar5B66mKCxd8AVPNGNe548PJl9FEW2pMF2P++4oQthu5vBDFx85h7ecvj/xBpObnPldxty4gn371ee0BwCDwTMoXKBupwJmbOwD5UE5ibxGNVj3n0YnrwCJ+56eUzibuCsLTI2/TvfJLp/nKKZtFFS5xmhvIGJR5+xT6+th8dG5DgpGGYJ36sTEwBcGBplSvqsF3iSb1dE8hpNN8/rTi43z5kPm4B90+17nS7LzvfLQozMChLru5xRD2WocwzkNUa9qST2qaJbkSvq9UMZepkKsuvC7qtoQhMesbwzFgTNEBsqHp61q6RIJYOh9enHjf6ejyE2YAksQLTOBK/hOp/RL1vk6f7lhpL/UX5lHVnE+qUZrQ8e+KQcfM=
-  file: vc
+  file:
+    - vc
+    - vault-client_1.1.2_amd64.deb
```

Comment:
I recommend `vault-client_*.deb`, because it's more generic and you don't want to change it with every version.